fee.ini

```diff
@@ -4,10 +4,10 @@
 
 [DEFAULT]
 # Fees and limits are chosen as upper value for successful transfer to manager.tz accounts
-fee=574
+fee=396
 
-[KTTX]
+[TZTX]
 # The storage limit corresponds to the amount needed to pay the burn fee for a 0 balance tz account
 # Storage limit has been reduced by a factor of 4 in Delphi (see https://blog.nomadic-labs.com/delphi-changelog.html)
```

Comment:
Can you update the comment as well please? It still refers to Delphi. A link to where the new values are taken from might be helpful)

Reply:
Comments have been revised.